### PR TITLE
refactor pfe-icon

### PR DESCRIPTION
**Pull request by mwcz**
This description got too long, extracted it into a blog post:

https://clayto.com/2019/07/web-component-icons/index.html

### CODE REVIEW

File: elements/pfe-icon/demo/pfe-icon.story.js [hunk 1]
```diff
@@ -1,85 +1,11 @@
 import { storiesOf } from "@storybook/polymer";
-import {
```
Comment (castastrophe) [inline elements/pfe-icon/demo/pfe-icon.story.js]: @mwcz I would hope we don't need to regress the story this much - can I jump in and help with re-implementing the attributes for styling?

File: elements/pfe-icon/package.json [hunk 2]
```diff
@@ -9,33 +9,24 @@
     "access": "public"
   },
   "description": "Icon element for PatternFly Elements",
-  "keywords": [
```
Comment (castastrophe) [inline elements/pfe-icon/package.json]: Why remove the keywords?

File: elements/pfe-icon/package.json [hunk 3]
```diff
@@ -9,33 +9,24 @@
...
-    "web-components",
-    "html"
-  ],
   "repository": {
     "type": "git",
-    "url": "github:patternfly/patternfly-elements",
-    "directory": "elements/pfe-icon"
+    "url": ""
```
Comment (castastrophe) [inline elements/pfe-icon/package.json]: Why regress the url paths and main locations?

File: elements/pfe-icon/package.json (hunk 2 above)
Comment (castastrophe) [inline elements/pfe-icon/package.json]: Why remove the keywords?

File: elements/pfe-icon/demo/pfe-icon.story.js (hunk 1 above)
Comment (mwcz) [reply elements/pfe-icon/demo/pfe-icon.story.js]: I haven't worked on the story at all yet.  I recreated the element from scratch so the previous icon's story was removed.  I've been pulling things back in from the previous implementation piece by piece, and haven't gotten to the story yet.

File: elements/pfe-icon/package.json (hunk 2 above)
Comment (mwcz) [reply elements/pfe-icon/package.json]: It's a brand new element; I'll pull in the keywords from the previous package.json.

File: elements/pfe-icon/package.json (hunk 3 above)
Comment (mwcz) [reply elements/pfe-icon/package.json]: I'll pull in those fields from the previous package.json.

File: elements/pfe-icon/package.json (hunk 2 above)
Comment (mwcz) [reply elements/pfe-icon/package.json]: Same as above.

File: elements/pfe-icon/demo/pfe-icon.story.js (hunk 1 above)
Comment (castastrophe) [reply elements/pfe-icon/demo/pfe-icon.story.js]: Oh I see!  That's great, let me know if I can help with anything.

File: elements/pfe-icon/package.json (hunk 2 above)
Comment (castastrophe) [reply elements/pfe-icon/package.json]: Sorry, it's confusing with the diffs showing up the way they are.  I'll remember that for next time!

**Pull request merged**